core/src/replay_stage.rs

```diff
@@ -2595,6 +2585,8 @@ impl ReplayStage {
                     bank_complete_time.as_us(),
                 );
                 execute_timings.accumulate(&r_replay_stats.execute_timings);
+
+                bank.read_cost_tracker().unwrap().report_stats(bank.slot());
```

Comment:
Would be good to grab read lock and execute slow `report_stats` in secondary thread. Like ` prioritization_fee_cache.finalize_priority_fee(bank.slot());` at Ln 2516. 

Or perhaps can keep `cost_update_service.rs` but shrink it down to calling `bank.read_cost_tracker().unwrap().report_stats(bank.slot());` in its service threads.

Reply:
i'm not sure this would add any benefit, the replay_stats reported in ln 2580, are probably as heavy.